### PR TITLE
ac-power: add simple getopt() argument parsing to systemd-ac-power

### PR DESCRIPTION
**Pull request by poettering**
We should probably do that for all our tools, hence fill in this gap
here.

### CODE REVIEW

File: src/ac-power/ac-power.c [hunk 1]
```diff
@@ -18,19 +18,93 @@
...
                 log_error_errno(r, "Failed to read AC status: %m");
-                return EXIT_FAILURE;
+                goto finish;
         }
 
-        return r != 0 ? EXIT_SUCCESS : EXIT_FAILURE;
+        if (arg_verbose)
+                puts(yes_no(r));
+
+        r = r ? EXIT_SUCCESS : EXIT_FAILURE;
```
Comment (keszybz) [inline src/ac-power/ac-power.c]: This line should be removed.

**Comment by keszybz** [comment]
RFE: man page

**Comment by poettering** [comment]
Not sure how I fucked that one up ;-)

Force pushed a new version now. PTAL.

**Comment by poettering** [comment]
Well, the binary is still in libdir, nor in bindir, let's not hurry too much with a man page ;-)

**Pull request merged**